fdbserver/DataDistribution.actor.cpp

```diff
@@ -925,8 +949,7 @@ ACTOR Future<Void> dataDistribution(Reference<DataDistributor> self,
 			state Reference<AsyncVar<bool>> processingWiggle(new AsyncVar<bool>(false));
 
 			if (SERVER_KNOBS->DD_TENANT_AWARENESS_ENABLED || SERVER_KNOBS->STORAGE_QUOTA_ENABLED) {
-				self->ddTenantCache = makeReference<TenantCache>(cx, self->ddId);
-				wait(self->ddTenantCache.get()->build());
+				wait(self->initTenantCache());
```

Comment:
Couldn't we get here even if DD is mocked? If so, we'll get an assertion failure in `initTenantCache()`.

Reply:
It's done in `initTenantCache()`